api/services/acquisition/src/providers/AcquisitionRepositoryProvider.ts

```diff
@@ -359,4 +359,41 @@ export class AcquisitionRepositoryProvider implements AcquisitionRepositoryProvi
     });
     return result.rows;
   }
+
+  async patchPayload<P = any>(
+    search: { operator_id: number; operator_journey_id: string; status: Array<AcquisitionStatusEnum> },
+    payload: P,
+  ): Promise<void> {
+    const connection = await this.connection.getClient().connect();
+    await connection.query('BEGIN');
+    try {
+      const oldPayloadResult = await connection.query({
+        text: `
+          SELECT payload FROM ${this.table}
+          WHERE operator_id = $1 AND journey_id = $2 AND status = ANY($3)
+          FOR UPDATE SKIP LOCKED
```

Comment:
Est-ce que ça veut dire que si une acquisition est locked par le process de traitement des acquistions, l'opérateur recevra un not found alors qu'en fait c'est juste un pb de concurrence ?